roles/etcd/meta/main.yml

```diff
@@ -17,4 +17,4 @@ galaxy_info:
   - system
 dependencies:
 - { role: os_firewall }
-- { role: etcd_common }
+- { role: etcd_server_certificates, etcd_cert_config_dir: /etc/etcd, etcd_cert_prefix: '', etcd_generated_certs_dir: /etc/etcd/generated_certs }
```

Comment:
This can actually be:
```
- role: etcd_server_certificates
  etcd_cert_config_dir: /etc/etcd
  etcd_cert_prefix: ''
  etcd_generated_certs_dir: /etc/etcd/generated_certs
```